zerver/openapi/zulip.yaml

```diff
@@ -14843,6 +14843,14 @@ paths:
                           exist as all onboarding steps were implicitly hotspots.
                         items:
                           $ref: "#/components/schemas/OnboardingStep"
+                      onboarding_video_url:
+                        type: string
+                        description: |
+                          URL of the onboarding video displayed to new users.
+
+                          Present unless disabled by the server administrators in `/etc/zulip/settings.py`.
```

Comment:
I think we probably want "Present if `onboarding_steps` is present in `fetch_event_types`." to be the behavior; i.e., key this under a `want("onboarding_steps")` check in `zerver/lib/events.py. 

(The alternative is `realm`, but given the coupling with onboarding, this seems right).

Reply:
Made the changes to make it present if `onboarding_steps` is present in `fetch_event_types`